### ISSUE TITLE
Mining Drone and Nothing else working

### ISSUE THREAD

**Issue by Yirok42** [mcenderdragon/Futurepack-API#786]
I wrote a question and now I can't find it. I have more information.

I am using Futurepack-1.16.4-31.1.8176

For the last two days I have had trouble where I have to exit the game to reset the ability to use the FP tools, techchests, techtables, etc anything to do with future pack.

I can only play for a few minutes and then nothing works.
I thought to stop my mining drone which is doing a 25x25 by about 60 mining task, and I was able to play longer (it still has not locked up)

Has anyone else seen this issue? I don't know that it is a bug or even that the mining drone is causing a problem yet. That is why I am asking this as a question.

**Comment by mcenderdragon** [comment]
looks like the internal server is lagging/crashing, can you provide logs?

**Comment by Yirok42** [comment]
I do not. I am not sure which logs are useful, and so at some point I deleted all the logs to see if when reproduced any new logs showed something which might look like an exception or logged error message. I still have this world and I was able to auto-mine again but I 1) moved the base cube, 2) am using smaller areas of mining. I have not reproduced the problem but I could try to mine a large area again to see if that is what broke it. I still suspect it is the mining drone as the only other thing i was doing was using the plasma cutter to acquire metal blocks from a bunker (that's what I call the room with the chests)
